docs/ops/logical/LogicalAnd_1.md

```diff
@@ -20,16 +20,16 @@
 
 **Inputs**
 
-* **1**: A tensor of type *T*. **Required.**
-* **2**: A tensor of type *T*. **Required.**
+* **1**: A tensor of type *T_BOOL*. **Required.**
```

Comment:
Make this also consistent to LogicalOr for both inputs:
A tensor of type \*T_BOOL\* and arbitrary shape

Reply:
Updated